### ISSUE TITLE
Question about Wan 2.2 High Noise Model Training

### ISSUE THREAD

**Issue by zhtjtcz** [modelscope/DiffSynth-Studio#728]
I used the Wan 2.2 full model training script in this repository and encountered the following problems during training:

When I train the Low Noise Model, the training results can be well adapted to the original High Noise Model, and the inference results are all normal.

But when I started training the High Noise Model, things changed. I found that the loss at this time is about dozens of times higher than the low training. At the same time, when I combined the trained High with the original Low, **the inference result would be a lot of noise**, as shown in the figure below:

<img width="966" height="1074" alt="Image" src="https://github.com/user-attachments/assets/5cf326c6-39d5-46fc-9ecf-e6bf2ae93e11" />

I'm not sure if anyone else has encountered similar problems. I guess training Lora should get the same conclusion as me. Is the training parameter configuration given by this repository the same as that used in the official training of Wan 2.2?

**Comment by Songssx** [comment]
I trained the i2v low-noise model of Wan2.2 using diffusion pipe. From the training results, it seems that the Lora model output after a certain number of steps produces a noisy video using the generated video.

**Comment by Songssx** [comment]
I seem to have to lower the shift value to something very small, like 0.5, before any characters appear, and even then it doesn't match the results I trained for.

**Comment by zhtjtcz** [comment]
> I trained the i2v low-noise model of Wan2.2 using diffusion pipe. From the training results, it seems that the Lora model output after a certain number of steps produces a noisy video using the generated video.

Have you tried training a low model with a full timestep? [Here](
https://github.com/tdrussell/diffusion-pipe/issues/332) is some discussion.

**Comment by Songssx** [comment]
I found the cause of the issue: I was using **a scaled quantized model** to train the LoRA, which was incorrect. After switching to **fp16** training, a new problem occurred. When I use the original **high noise + (LoRA + low noise)** combination to generate videos, the LoRA has almost no effect. Only when I use **low noise + (LoRA + low noise)** or **(low noise + LoRA) + (LoRA + low noise)** does the output meet the expected results.

I’m not sure how to use **diffusion-pipe** to train **a shared LoRA model** for both **Wan2.2 models (high noise and low noise)**.

**Comment by Artiprocher** [comment]
@Songssx @zhtjtcz Thank you for the feedback. We have observed a similar issue, and we will conduct some experiments for verification and update our code accordingly.

**Comment by boqian-li** [comment]
@Artiprocher, hi, has this problem been solved?

**Comment by SummerTrains** [comment]
> Wan 2.2 full model training script

where is the Wan 2.2 full model training script ?
